### PR TITLE
Added tests for PropertiesLoader

### PR DESCRIPTION
**Pull request by rchakra3**
@mishako I'd appreciate a review!

Coverage Improvements:
- Line Coverage 69% -> 82%
- Mutation Coverage 40% -> 75%

In addition, I realized that [PropertiesLoader](https://github.com/rometools/rome/blob/master/rome/src/main/java/com/rometools/rome/io/impl/PropertiesLoader.java) does not explicitly handle the case of a missing plugins file (A nonexistent plugins file would throw a NullPointerException with the current code). Is this by design?

### CODE REVIEW

**Comment by mishako** [comment]
@rchakra3 You mean `masterFileLocation` argument in `PropertiesLoader` constructor? I guess it is by design that the master file must exist.

File: rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import static org.junit.Assert.*;
+
+public class PropertiesLoaderTest {
+
+    private PropertiesLoader propertyLoader;
+
+    @Before
+    public void setUp() throws Exception {
+        Constructor[] ctors = PropertiesLoader.class.getDeclaredConstructors();
```
Comment (mishako) [inline rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java]: Better to make the constructor package-private?

File: rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        Constructor ctor = ctors[0];
+        ctor.setAccessible(true);
+        propertyLoader = (PropertiesLoader) ctor.newInstance("rome.properties", "test.rome.properties");
```
Comment (mishako) [inline rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java]: `rome.properties` seems to be used by another test. Maybe create your own file?

File: rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Test
+    public void testPropertyListLength() {
+        String propertyList[] = propertyLoader.getProperty("test_property");
+        assertEquals(1, propertyList.length);
+    }
+
+    @Test
+    public void testPropertyValue() {
```
Comment (mishako) [inline rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java]: Merge `testPropertyValue` with `testPropertyListLength` and use a single assert: `assertArrayEquals()`?

File: rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        String propertyList[] = propertyLoader.getProperty("test_property");
+        assertEquals("part1 part2", propertyList[0]);
+    }
+
+    @Test
+    public void testPropertyValueTokenized() {
+        String tokenizedProperties[] = propertyLoader.getTokenizedProperty("test_property", " ");
+        assertEquals(2, tokenizedProperties.length);
+        assertEquals("part1", tokenizedProperties[0]);
+        assertEquals("part2", tokenizedProperties[1]);
```
Comment (mishako) [inline rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java]: `assertArrayEquals()`?

File: rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    }
+}
```
Comment (mishako) [inline rome/src/test/java/com/rometools/rome/io/impl/PropertiesLoaderTest.java]: Maybe also test that it loads properties from both files? And what happens when there are duplicate properties?

File: rome/src/test/resources/test.rome.properties [hunk 6]
```diff
@@ -0,0 +1 @@
+test_property=part1 part2
```
Comment (mishako) [inline rome/src/test/resources/test.rome.properties]: Maybe put test resources in the same package, i.e. `com.rometools.rome.io.impl`?

**Comment by mishako** [comment]
@rchakra3 Any thoughts?

**Comment by rchakra3** [comment]
@mishako Sorry, I somehow completely missed this. I'll work on your comments and update the PR by the end of the week :)

**Comment by mishako** [comment]
Closing this PR since it's almost 1 year old now.
